empress/__init__.py

```diff
@@ -197,6 +210,7 @@ def draw_on(self, axes: plt.Axes, log=False):
                                   self._dup_min, self._dup_max, log=False)
 
 
+
```

Comment:
Remove this new line

Reply:
Just reset this file to the one in master -- https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision

Reply:
Removed.